internal/component/database_observability/mysql/component.go

```diff
@@ -311,6 +312,24 @@ func (c *Component) startCollectors() error {
 		c.collectors = append(c.collectors, qsCollector)
 	}
 
+	if collectors[collector.SetupConsumersName] {
+		scCollector, err := collector.NewSetupConsumer(collector.SetupConsumerArguments{
+			DB:             dbConnection,
+			Registry:       c.registry,
+			Logger:         c.opts.Logger,
+			ScrapeInterval: 1 * time.Hour,
```

Comment:
Scrape interval is hardcoded at the moment. Should we make this configurable (separate from the other collect intervals)?